amr-wind/projection/incflo_apply_nodal_projection.cpp

```diff
@@ -247,7 +250,7 @@ void incflo::ApplyProjection(
     // Create sigma while accounting for mesh mapping
     // sigma = 1/(fac^2)*J * dt/rho
     Vector<amrex::MultiFab> sigma(finest_level + 1);
-    if (variable_density || mesh_mapping) {
+    if (variable_density || mesh_mapping || is_anelastic) {
```

Comment:
based on what you have in PDEBase.cpp, I think the "|| is_anelastic" doesn't need to be here, because anelastic is going to force variable_density = true.

Reply:
yup that's a holdover. good catch